packages/node_modules/nav-frontend-alertstriper/src/alertstripe.js

```diff
@@ -8,13 +8,15 @@ import 'nav-frontend-alertstriper-style'; // eslint-disable-line import/extensio
 const cls = (type, solid, className) => classNames('alertstripe', className, {
     'alertstripe--solid': solid,
     'alertstripe--advarsel': type === 'advarsel',
+    'alertstripe--feil': type === 'feil',
```

Comment:
Er ikke dette bare en fylt versjon av `advarsel`-typen?

Reply:
Svar fra Sergio: 
"Hei, 

Riktig, det en "ikke fylt" versjon av alertstripe-advarsel ikon er annerldes og kan anses mer som en “stop” type melding. Men det er egentlig eneste forskjell. Jeg had nok følgt navn konvensjon vi har brukt tidligere men kanskje tilsette en parametre for stop ikonen (om det lar seg gjøre og er fornuftig). "